recipe/meta.yaml

```
@@ -7,41 +7,49 @@ package:
 
 source:
   url: https://github.com/pymc-devs/{{ name }}/archive/refs/tags/rel-{{ version }}.tar.gz
-  sha256: 543eebd466db93190ac454c5a633948f724a64021e4fe39d89344e7423a5798b
+  sha256: 34210e120808f811b582ff1963667590f67b04af2a89bcdb611cd1438bf4c41b
+  patches:
+    - 0001-patch-pyproject.toml.patch
 
 build:
-  number: 1
+  number: 0
   script:
     - python -m pip install . -vv --no-deps --no-build-isolation
   entry_points:
     - pytensor-cache = pytensor.bin.pytensor_cache:main
-  skip: True #  [py<=310]
+  # upstream requirement
+  skip: True  # [py<310 or py>=314]
+  ignore_run_exports:
+    - libstdcxx-ng
+    - numpy
 
 requirements:
   build:
     - git  # [not win]
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
+    - patch     # [not win]
+    - m2-patch  # [win]
   host:
     - python
     - cython
     - pip
     - numpy {{ numpy }}
-    - versioneer 0.28
+    - versioneer 0.29
```

https://github.com/pymc-devs/pytensor/blob/rel-2.31.3/pyproject.toml#L6
```suggestion
    # versioneer[toml]==0.29
    - versioneer 0.29
    - toml
```